src/Builder/CliMenuBuilder.php

```diff
@@ -560,6 +572,18 @@ private function propagateStyles(CliMenu $menu, array $items = [])
         $currentItems = !empty($items) ? $items : $menu->getItems();
 
         foreach ($currentItems as $item) {
+            if ($item instanceof CheckboxItem
+                && !$item->getStyle()->hasChangedFromDefaults()
+            ) {
+                $item->setStyle(clone $menu->getCheckboxStyle());
+            }
+
+            if ($item instanceof RadioItem
+                && !$item->getStyle()->hasChangedFromDefaults()
+            ) {
+                $item->setStyle(clone $menu->getRadioStyle());
+            }
```

Comment:
I don't like this and suspect it will get uglier. But also I'm happy for us to fix this later.

Reply:
I mean refactor, just because of all the `instanceof` checks.